configs/localization/ssn/ssn_r50_1000e_anet_rgb_train.py

```diff
@@ -0,0 +1,155 @@
+ann_file_train = 'anet_val_formatted_proposal.txt'
+ann_file_val = 'anet_val_formatted_proposal.txt'
+ann_file_test = 'anet_val_formatted_proposal.txt'
+img_norm_cfg = dict(mean=[104, 117, 128], std=[1, 1, 1], to_bgr=True)
+train_pipeline = [
+    dict(
+        type='SampleProposalFrames',
+        clip_len=1,
+        body_segments=5,
+        aug_segments=(2, 2),
+        aug_ratio=0.5),
+    dict(type='RawFrameDecode'),
+    dict(type='Resize', scale=(340, 256), keep_ratio=True),
+    dict(type='CenterCrop', crop_size=224),
+    dict(type='Flip', flip_ratio=0),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='FormatShape', input_format='NPTCHW'),
+    dict(
+        type='Collect',
+        keys=[
+            'imgs', 'reg_targets', 'proposal_scale_factor', 'proposal_labels',
+            'proposal_type'
+        ],
+        meta_keys=[]),
+    dict(
+        type='ToTensor',
+        keys=[
+            'imgs', 'reg_targets', 'proposal_scale_factor', 'proposal_labels',
+            'proposal_type'
+        ])
+]
+val_pipeline = [
+    dict(
+        type='SampleProposalFrames',
+        clip_len=1,
+        body_segments=5,
+        aug_segments=(2, 2),
+        aug_ratio=0.5),
+    dict(type='RawFrameDecode'),
+    dict(type='Resize', scale=(340, 256), keep_ratio=True),
+    dict(type='CenterCrop', crop_size=224),
+    dict(type='Flip', flip_ratio=0.5),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='FormatShape', input_format='NPTCHW'),
+    dict(
+        type='Collect',
+        keys=[
+            'imgs', 'reg_targets', 'proposal_scale_factor', 'proposal_labels',
+            'proposal_type'
+        ],
+        meta_keys=[]),
+    dict(
+        type='ToTensor',
+        keys=[
+            'imgs', 'reg_targets', 'proposal_scale_factor', 'proposal_labels',
+            'proposal_type'
+        ])
+]
+data = dict(
+    videos_per_gpu=1,
+    workers_per_gpu=2,
+    train=dict(
+        type=dataset_type,
+        ann_file=ann_file_train,
+        data_prefix=data_root,
+        train_cfg=train_cfg,
+        test_cfg=test_cfg,
+        body_segments=5,
+        aug_segments=(2, 2),
+        aug_ratio=0.5,
+        test_mode=False,
+        verbose=True,
+        pipeline=train_pipeline),
+    val=dict(
+        type=dataset_type,
+        ann_file=ann_file_val,
+        data_prefix=data_root,
+        train_cfg=train_cfg,
+        test_cfg=test_cfg,
+        body_segments=5,
+        aug_segments=(2, 2),
+        aug_ratio=0.5,
+        test_mode=False,
+        pipeline=val_pipeline))
+# optimizer
+optimizer = dict(
+    type='SGD', lr=0.001, momentum=0.9,
+    weight_decay=1e-6)  # this lr is used for 8 gpus
+optimizer_config = dict(grad_clip=dict(max_norm=35, norm_type=2))
+# learning policy
+lr_config = dict(policy='step', step=[200, 400, 600, 800])
+checkpoint_config = dict(interval=5)
+log_config = dict(interval=1, hooks=[dict(type='TextLoggerHook')])
+# runtime settings
+total_epochs = 1000
```

Comment:
1000 epochs seem to be a quite large number. How long does it take to train the SSN for 1000 epochs on ActivityNet?

Reply:
About 2 weeks.